packages/www/docs/guides/upload-a-video/index.mdx

```diff
@@ -0,0 +1,775 @@
+---
+title: Upload a Video
+description: Livepeer's Upload a Video Guide
+metaTitle: Upload a Video Guide - Livepeer.com
+metaDescription: Livepeer's Upload a Video Guide
+---
+
+# Upload a Video
+
+This tutorial covers how to upload the video to Livepeer to allow viewers to
+playback stored video assets in your application.
+
+## Caveats
+
+- Have to manually check the status of the file using `List Task` method to see
+  if it has finished uploading.
+- File size is limited to 1GB. Any files greater than 1GB will error with
+  NETWORK CHANGED.
```

Comment:
I'm not aware of this error and do not believe that it will always error out with that. Would rephrase with
```suggestion
- Files are currently limited to 1GB in size. Any files greater than that will likely error out during the upload or processing steps.
```